roles/telemetria/README.md

```diff
@@ -0,0 +1,53 @@
+# Telemetry
+
+## About
+
+This readme contains the necessary steps to deploy  [`Grafana`](https://grafana.com/grafana/) for telemetry using keycloak for authentication
+
+## Disclaimer
+
+With a focus on improving the service, aiming at greater maintainability and programmability of the solution. We started a refactoring of the service using [`Helm`](https://helm.sh/), initially we are porting grafana and then we will porte all the rest of the observability solution.
+
+As the inclusion of services (Prometheus, Loki, Promtail and Grafana) is under development, some services are currently not being monitored. The following describes the services already implemented.
```

Comment:
Aqui está falando "The following describes the services already implemented." porém nas linhas abaixo não tem uma lista com os serviços que já podem ser monitorados.

Reply:
Peguei como base o Readme do monitoring e essa parte realmente não faz sentido para telemetria, irei retirar